modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json

```diff
@@ -7,6 +7,30 @@
 		"en_US": "MDF Claim"
 	},
 	"name": "MDFClaim",
+	"objectActions": [
```

Comment:
@eliasantosf what does this action do? is it just a placeholder to add the logic later? also it may be helpful to make the externalReferenceCode something human readable, like the other notification action ercs

Reply:
@ryanschuhler, as soon as the client extensions for the complete status of the request are in the master, I'll call it in the `objectActionExecutorKey`. I've already made the action definition ready for this, which is the case.

Reply:
great, makes sense, thanks!